Lib/test/test_genericalias.py

```diff
@@ -35,7 +36,7 @@ def test_subscriptable(self):
                   Mapping, MutableMapping, MappingView,
                   KeysView, ItemsView, ValuesView,
                   Sequence, MutableSequence,
-                  MappingProxyType, DirEntry
+                  MappingProxyType, DirEntry, _BaseNetwork
```

Comment:
I'd put a newline here -- let's only group names together that were imported from the same module.

Also, perhaps the test should check that the *public* classes defined in ipaddress are generic, rather than the private base class.

Finally, shouldn't `_BaseInterface` also be made generic? It is generic in typeshed.

Reply:
`_BaseInterface` doesn't look like a real class

Reply:
IPv4/v6Interface is now supported and rest of your suggestions applied.